python/ray/__init__.py

```diff
@@ -155,7 +155,11 @@ def _configure_system():
 # some functions in the worker.
 import ray.actor  # noqa: E402,F401
 from ray.actor import method  # noqa: E402
-from ray.cross_language import java_function, java_actor_class  # noqa: E402
+from ray.cross_language import (
+    java_function,
+    java_actor_class,
+    cpp_function,
```

Comment:
We should consider keeping xlang functionality in the package to avoid polluting the top-level Ray namespace.

Reply:
This API had introduced a breakage for a large number of our applications which have been running for a long time. 
I think we should mark it as deprecating and then remove it in 2.0. CC @raulchen